api/v1alpha1/operator_labels.go

```diff
@@ -26,6 +26,9 @@ const (
 	// WatchedByLabel defines a redirect to a controller that should be getting a notification
 	// if this resource is changed.
 	WatchedByLabel = OperatorPrefix + Separator + "watched-by"
+	// PurposeLabel defines the purpose of the resource, i.e. Secrets which will be used to certificate management.
+	PurposeLabel = OperatorPrefix + Separator + "purpose"
```

Comment:
This is used to filter the Issuer ([here](https://github.com/jeremyharisch/kyma-operator/blob/30eec89c4ce5654443612835a66b6f493c7d8a33/pkg/watcher/certificate.go#L247))

Since I did not find another label which can be reused for such purpose, I thought of introducing this new one.
Labels are set [here](https://github.com/jeremyharisch/kyma-operator/blob/ebccd8005142697a05494d62b64dd83d3ed69eaa/config/watcher/certificate_setup.yaml#L18)

Meaning, the Issuer will have the Operator Name label and a purpose label. We could also remove the purpose label and just keep the `managed-by`, but then we always need to make sure we will only have this single Issuer

Reply:
Furthermore, this label will be set to the secrets the cert-manager will create - just for a better understanding why such secrets are being created